### PR TITLE
SF-3452 rescue ata funds

### PR DESCRIPTION
**Pull request by thesoftwarejedi**

### CODE REVIEW

**Review by nicholas-ewasiuk — APPROVED** [review]
Makes sense to me

**Review by JamesMDunn — APPROVED** [review]

**Comment by thesoftwarejedi** [comment]
One more approval req'd.

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,46 +3,32 @@ version: 2.1
...
+      - image: backpackapp/build:v0.28.0
     steps:
       - checkout
-      - run:
-          name: "Unit tests"
-          command: "cargo test --lib"
       - run:
           name: "Anchor tests"
           command: |
+            npm i
```
Comment (quellen-sol) [inline .circleci/config.yml]: `yarn install`

File: package-lock.json
Comment (aaronovz1) [inline package-lock.json:1]: Why was lock file deleted instead of updated?

Comment (thesoftwarejedi) [reply package-lock.json:1]: anchor upgrade - needed it to rejigger the refs completely.  node only used for tests anyhow.

**Review by quellen-sol — APPROVED** [review]
LFG

**Pull request merged**